erpnext/manufacturing/doctype/bom/bom.json

```
@@ -506,13 +507,19 @@
    "label": "Manufacturing Type",
    "options": "\nDiscrete\nProcess",
    "reqd": 1
+  },
+  {
+   "fieldname": "avg_manufactured_qty",
+   "fieldtype": "Float",
+   "label": "Avg Manufactured Qty",
```

```suggestion
   "label": "Average Manufactured Quantity",
```
label should not be shortened, it wont be translated properly